.github/workflows/rhub.yaml

```
@@ -96,12 +96,13 @@ jobs:
         with:
           job-config: ${{ matrix.config.job-config }}
 
-      - uses: insightsengineering/setup-r-dependencies@v1
+      - uses: insightsengineering/setup-r-dependencies@install_quarto_param
```

revert after merge of https://github.com/insightsengineering/setup-r-dependencies/pull/12

```suggestion
      - uses: insightsengineering/setup-r-dependencies@main
```